### ISSUE TITLE
Missing Alt attribute in ArticleImage class

### ISSUE THREAD

**Issue by thuongmhh** [nozzlegear/ShopifySharp#1058]
According to [Shopify REST API](https://shopify.dev/docs/api/admin-rest/2024-01/resources/article#put-blogs-blog-id-articles-article-id), and article image should have 3 properties: attachment, src, and alt.

The alt property is currently missing from the ArticleImage class, making it difficult to update the alt text for an article image.

Please help adding that property in the next release.
